wiki/Gameplay/Game_modifier/Relax/de.md

```diff
@@ -0,0 +1,56 @@
+---
+stub: true
+tags:
+  - relax
+  - RL
+  - RX
+  - mod
+  - game modifier
+  - Spielmodifikation
+---
+
+# Relax (Mod)
+
+![RL Modsymbol](/wiki/shared/mods/RL.png "Relax (RL) Modsymbol")
+
+*Für die vollständige Liste aller Mods, siehe: [Spielmodifikationen](/wiki/Game_modifier)*
+
+## Übersicht
+
+- Abkürzung: RL oder RX
+- Typ: Sonstige
+- Score-Multiplikator: 0,00x
+- Standard-Tastenkürzel: `Z`
+- Untertitel:
+  - ![][osu!]: `Du musst nicht mehr klicken. Gönne deinen Fingern eine Pause. ** UNGEWERTET **`
+  - ![][osu!taiko]: `Sanfte Gemüter können sich erholen von blitzschnellen Spinnern, anspruchsvollen Drummrolls und unerwarteten Katus. ** UNGEWERTET **`
+  - ![][osu!catch]: `Benutze die Maus, um den Ryuuta zu bewegen. ** UNGEWERTET **`
+- Kompatible Spielmodi: ![][osu!] ![][osu!taiko] ![][osu!catch]
+
+## Beschreibung
+
+*Hinweis: Das Aktivieren der Mod Relax sorgt dafür, dass das Spiel nicht gerankt wird und nicht in der lokalen Rangliste gespeichert wird.*
+
+Die Mod **Relax** ist eine [Spielmodifikation](/wiki/Game_modifier), die das Ziel hat, Spielern ein entspanntes Spielerlebnis zu bieten. Aufgrund ihrer Funktionalität wird die Mod typischerweise verwendet, um schwierige [Patterns](/wiki/Beatmap/Pattern) auf [Beatmaps](/wiki/Beatmap) zu analysieren/üben oder um Spielern zu ermöglichen, der Hand, die zum Tippen benutzt wird, eine Pause zu gönnen.
+
+### osu!
+
+In [osu!](/wiki/Game_mode/osu!) muss der Spieler beim Aktivieren der Mod Relax nur den Cursor über [Hit-Objekte](/wiki/Hit_object) bewegen. Alle Hit-Objekte werden automatisch von osu! getippt. Die Mod macht es in allen Spielmodi unmöglich, die Beatmap nicht zu schaffen.
+
+Die Mod Relax deaktiviert auch die Benutzeroberfläche im Spiel, wodurch der [Score](/wiki/Gameplay/Score), die [Combo](/wiki/Beatmapping/Combo) und Misses versteckt bleiben. Das kann jedoch über `Shift` + `Tab` wieder aktiviert werden. Zusätzlich werden die Spins pro Minute (SPM) verdoppelt, was dem Spieler ermöglicht, viel schneller zu drehen und so einen höheren Score zu erzielen als in einem gewöhnlichen Spiel der Fall wäre.
+
+### osu!taiko
+
+In [osu!taiko](/wiki/Game_mode/osu!taiko) entfernt die Mod Relax die Beurteilung der Farbe: Die richtige Farbe für die Note wird unabhängig von der eingegebenen Farbe getroffen. Die Farben der Noten ändern sich allerdings nicht.
+
+Dazu kommt, dass die Mod Relax ebenfalls ein größeres Zeitfenster zulässt, um Sliderticks während Drumrolls zu sammeln. Dendens können mit jeder Farbsequenz absolviert werden anstatt die Farbsequenz abzuwechseln. Das animierte Maskottchen wird entfernt.
+
+Im Gegensatz zu osu! *zeigt* die Mod Relax in osu!taiko den Score, die Combo und Misses an.
+
+### osu!catch
+
+In [osu!catch](/wiki/Game_mode/osu!catch) erlaubt die Mod Relax, dass sich der Catcher frei bei jeder Geschwindigkeit mit der Maus oder anderen Peripheriegeräten zur Steuerung des Cursors bewegen kann.
```

Comment:
The original article uses `character` here. At first I thought about using `Spielfigur`, but using the actual name `Catcher` is fine too in my opinion.